packages/aragon-wrapper/src/index.js

```diff
@@ -772,6 +774,29 @@ export default class Aragon {
     return this.accounts.pipe(first()).toPromise()
   }
 
+  /**
+   * Allows apps to sign arbitrary data via a RPC call
+   *
+   * @param {Array<Object>} params An object containing the address of the app and the message to be signed
+   * @return {Promise<string>} signature hash
+   */
+  signMessage (params) {
```

Comment:
>  However, your comment here says we should only need one parameter on the aragon/wrapper method

Ahh, yeah, actually I think it's better to go back to the two-param method instead (`signMessage(message, fromApp)`). Hadn't considered that this required the originating app when I first reviewed.

Sorry for the churn 🙏!

Reply:
No worries! Is the variable name `appAddress` ok? Or would you like `fromApp`? Wondering bc i'll need to make the updates in https://github.com/aragon/aragon/pull/683 as well

Reply:
I kept as `appAddress` for now, but if you would like me to change it that's no problem.

Reply:
Actually, maybe `requestingApp`?

Just trying to find something descriptive 😄

Reply:
hmmm yeah. I chose `appAddress` because it's used by the permissions manager in aragon/wrapper